dockerized

hector-baez opened the pull request:
Added missing R/apt dependencies
Hosted on dockerhub: https://hub.docker.com/repository/docker/hbaez/reports/general

hector-baez committed:
add dependencies

hector-baez committed:
rm flask steps

jimjeffers commented:
@hector-baez question - do you know if we have any deps in the conda env that we can remove because they are things we're not using anymore. I see plumbr in there as one example. Also are all of the `apt-get install -y` deps necessary to install the conda dependencies? I just want to make sure we're not installing the same dependencies twice if we ultimately are relying on the conda environment.

hector-baez committed:
remove unused dependencies

hector-baez commented:
@jimjeffers I removed the unused conda dependencies and removed postgresql from the apt-get command. The building time is still ~1hr, with most of the time running `install.R`.

jimjeffers commented:
Awesome. Oh well at least it's as lean as we can get this I suppose. Thanks for getting this working!
